engines/mohawk/dialogs.cpp

```diff
@@ -119,6 +120,17 @@ MystOptionsWidget::MystOptionsWidget(GuiObject *boss, const Common::String &name
 		                                             _("The Myst fly by movie was not played by the original engine."));
 	}
 
+	if (!isDemo) {
+/* I18N: 
+This Option is for hard-of-hearing.
+It makes it easier to solve the spaceship puzzle.
+Normally game uses strict binary logic here.
+We change it to use fuzzy logic.
+*/
```

Comment:
Hello,
I'm sorry but I'm not sure I understand your comment here. You mean I have to indent the whole comment so that it is in line with rest of the code inside of the { } ?

Reply:
I changed spacing. If this is what you wanted then it can be marked Resolved.